test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -490,7 +493,17 @@ def run(self, test):
         self.result = result = self._makeResult()
         self.result.start = datetime.datetime.now()
         try:
-            test(result)
+            for i in range(self.repetition):
+                self.logger.info("====================================")
+                self.logger.info("tests are running (" + str(i + 1) + ") time")
+                self.logger.info("====================================")
+                if i is not 0:
+                    time.sleep(self.repetition_delay)
+                test(result)
+            self.logger.info("=============================================")
+            self.logger.info(" All tests are repeated " +
```

Comment:
`self.logger.info("All Tests are repeated %d times" % self.repeat_count)`

Also shouldn't we print this message only if repeat count is > 0?

Reply:
Actually repeat_count = 1 by default, so it should be > 1.